src/main/java/se/sundsvall/casemanagement/integration/ecos/RiskClassService.java

```diff
@@ -34,164 +34,171 @@
-                .toList());
-    }
-
-    private ArrayOfSaveRiskClass2024CertificationDto mapThirdPartyCertifications(String thirdPartyCertString) {
-        if (thirdPartyCertString == null) {
-            return new ArrayOfSaveRiskClass2024CertificationDto().withSaveRiskClass2024CertificationDto(new SaveRiskClass2024CertificationDto());
-        }
-        var dtos = splitString(thirdPartyCertString);
-
-        return new ArrayOfSaveRiskClass2024CertificationDto()
-            .withSaveRiskClass2024CertificationDto(
-                dtos.stream()
-                    .map(dto -> new SaveRiskClass2024CertificationDto()
-                        .withThirdPartyCertificationText(dto))
-                    .toList());
-    }
-
-    private List<String> splitString(String string) {
-        return Arrays.stream(string.split(" "))
-            .map(s -> s.split(","))
-            .flatMap(Arrays::stream)
-            .toList();
-    }
+	private static final String MAIN_ORIENTATION_ID = "MainOrientationId";
+
+	private static final String PROD_SIZE_ID = "ProductionSizeId";
+
+	private static final String IS_SEASONAL = "IsSeasonal";
+
+	private static final String SEASONAL_NOTE = "seasonalNote";
+
+	//Since there is too many activities for one extra parameter (on the openE side)
+	// we need to split it up in different extra parameters.
+	private static final List<String> ACTIVITIES = List.of("activities", "activities2", "activities3", "activities4");
+
+	private static final String PRODUCT_GROUPS = "productGroups";
+
+	private static final String THIRD_PARTY_CERTS = "thirdPartyCertifications";
+
+	private final MinutMiljoClient minutMiljoClient;
+
+	public RiskClassService(final MinutMiljoClient minutMiljoClient) {
+		this.minutMiljoClient = minutMiljoClient;
+	}
+
+	public void updateRiskClass(final EnvironmentalCaseDTO caseInput, final String caseId) {
+		final var facilityId = searchFacility(extractOrgNr(caseInput), caseInput.getFacilities().get(0).getFacilityCollectionName());
+		addFacilityToCase(facilityId, caseId);
+		final var data = createSaveRiskClassObject(facilityId, caseId, caseInput);
+		minutMiljoClient.updateRiskClass(data);
+	}
+
+	private String extractOrgNr(final EnvironmentalCaseDTO eCase) {
+		return CaseUtil.getSokigoFormattedOrganizationNumber(eCase.getStakeholders().stream()
+			.map(stakeholderDTO -> {
+				if (stakeholderDTO instanceof final OrganizationDTO orgDTO) {
+					return orgDTO.getOrganizationNumber();
+				} else return "";
+			})
+			.findFirst()
+			.orElse(""));
+	}
+
+	private String searchFacility(final String orgNr, final String facilityCollectionName) {
+
+
+		final var facilityTypeFilter = new FacilityFacilityTypeIdsFilterSvcDto()
+			.withFacilityTypeIds("4958BC00-76E8-4D5B-A862-AAF8E815202A");
+
+		final var facilityStatusFilter =
+			new FacilityFacilityStatusIdsFilterSvcDto()
+				.withFacilityStatusIds(new ArrayOfguid().withGuid(
+					"88E11CAA-DF35-4C5E-94A8-3C7B0369D8F2", //Anmäld/Ansökt
```

Comment:
Are these ID:s the same across environments (E.g. test/prod)?
If not: Should they be externalized (property-files)?

I think that they at least should be put in a variable for increased readability, e.g.:

```
private static final String FACILITY_STATUS_ID_APPLIED = "88E11CAA-DF35-4C5E-94A8-3C7B0369D8F2";
private static final String FACILITY_STATUS_ID_INACTIVE = "64B2DB7A-9A11-4F20-A57C-8122B1A469E6;
// ...and so on.
```

Reply:
They are the same. (This class is only hit with the auto-formatter and the logic is in production. Just FYI) But I'll move the GUIDs to static vars as I agree